core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.spec.ts

```diff
@@ -127,6 +131,26 @@ describe('Story node editor directive', function() {
         return deferred.promise;
       }
     };
+
+    var MockSkillBackendApiService = {
+      fetchMultiSkillsAsync: (skillIds) => {
+        if (skillIds[0] === '2') {
+          return $q.reject();
```

Comment:
Add a comment explaining why this is rejected.

Reply:
Done